src/Library/demos/Popovers/main.blp

```diff
@@ -0,0 +1,91 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: "Popovers";
+  description: _("Display content in a container above all other content on the window");
+  Box {
+    orientation: vertical;
+    halign: center;
+
+
+    Box {
+      margin-bottom: 80;
+      margin-top: 40;
+
+      MenuButton popover_button {
+      name: 'popover_button';
+      popover: plain_popover;
+      label: "plain popover";
+      primary: true;
+      margin-end: 40;
+      }
+
+      MenuButton popover_menu_button {
+        name: "popover_menu_button";
+        label: "Popover menu";
+        margin-end: 40;
+        popover: popover_menu;
+      }
+    }
+
+    Label sub_heading {
+      label: "API References";
+      styles ["title-4"]
+    }
+
+    LinkButton {
+      label: "Popover";
+      uri: "https://docs.gtk.org/gtk4/class.Popover.html";
+    }
+
+    LinkButton {
+      label: "Popover menu";
+      uri: "https://docs.gtk.org/gtk4/class.PopoverMenu.html";
+    }
+
+    LinkButton {
+      label: "Popover menu bar";
+      uri: "https://docs.gtk.org/gtk4/class.PopoverMenuBar.html";
+    }
+
+    LinkButton {
+      margin-top: 20;
```

Comment:
please use only multiples of 6

it's a GNOME design guideline

 - `24` works well here

Reply:
there are still spacings that are not multiples of 6

Reply:
Oh sorry! I missed this